### PR TITLE
lapack: update 3.6.0

### PR DESCRIPTION
**Pull request by hyuni**
how to test library?!

### CODE REVIEW

File: lapack.rb [hunk 1]
```diff
@@ -1,26 +1,19 @@
...
   homepage "http://www.netlib.org/lapack/"
-  url "http://www.netlib.org/lapack/lapack-3.5.0.tgz"
-  sha256 "9ad8f0d3f3fb5521db49f2dd716463b8fb2b6bc9dc386a9956b8c6144f726352"
-
-  bottle do
-    cellar :any
-    sha256 "b492305db3e74f7fde1d1798fbbb653c9caea0f7436e8bda5af2c19677909fc0" => :yosemite
-    sha256 "c099b310cbe3217266a9316cff77471c1c72ad0203def9776c851609e38ea789" => :mavericks
-    sha256 "cd6aea6fb9bc22942172bb532a0e6836320b28406341985a11fe9d1dd9ee62ff" => :mountain_lion
-  end
```
Comment (bfontaine) [inline lapack.rb]: Please don’t remove this block; it’ll be updated by our CI bot. Thanks!

**Comment by bfontaine** [comment]
See e.g. [`tinyxml2.rb`](https://github.com/Homebrew/homebrew/blob/422e87910fef79f762a5d0aa6eaa95e47d014e46/Library/Formula/tinyxml2.rb#L23-L33) for an example of a library formula with a good test.

File: lapack.rb (hunk 1 above)
Comment (hyuni) [inline lapack.rb]: @bfontaine thanks. asap update PR

**Comment by hyuni** [comment]
@bfontaine thanks. updated PR.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew, @hyuni; we appreciate it! :tada:

**Pull request closed without merging**